### DISCUSSION TITLE
Security Onion does not work after update to 2.3.100

### DISCUSSION THREAD

**Discussion by YannickZ5** [Security-Onion-Solutions/securityonion#7049] [Unsupported Versions] [answered]
Hello everyone,

after my update from 2.3.91 to 2.3.100 security onion does not work anymore because no service is starting.

here are some information about my deployment:

Version: 2.3.100
Deployment: on-prem
Installation: ISO /Standalone
Does so-status show all services running? No

- Only Docker, so-aptcacherng and so-kratos are running. All the other services displaying "WAIT_START"

Do you get any failures when you run sudo salt-call state.highstate? - Yes
```

local:
    Data failed to compile:
----------
    Rendering SLS 'base:logstash' failed: Jinja variable 'None' has no attribute 'items'
/var/cache/salt/minion/files/base/logstash/map.jinja(7):
---
[...]
{% set REDIS_NODES = [] %}
{% set mainint = salt['pillar.get']('host:mainint') %}
{% set localhostip = salt['grains.get']('ip_interfaces').get(mainint)[0] %}
{% if role in ['so-node', 'so-standalone', 'so-managersearch'] %}
  {% set node_data = salt['pillar.get']('logstash:nodes') %}
  {% for node_type, node_details in node_data.items() | sort %}    <======================
    {% if node_type in ['manager', 'managersearch', 'standalone', 'receiver' ] %}
      {% for hostname in node_data[node_type].keys() %}
        {% do REDIS_NODES.append({hostname:node_details[hostname].ip}) %}
      {% endfor %}
    {% endif %}
[...]
---
```

Does the SOC Grid page show any failures? - I don't can connect to the Web-Frontend

Thank you for any help!

**Comment by m0duspwnens** [comment, marked as answer]
Can you post the soup log?

**Reply by YannickZ5** [reply]
Sorry I can't. I had to bring up a backup from SO as fast as possible because of several reasons. At the moment I have no opportunity to investigate on the failed system.

**Reply by TOoSmOotH** [reply]
Not much we can help with if you don't have the log.
